build.gradle.kts

```diff
@@ -58,6 +60,8 @@ tasks.named<ShadowJar>("shadowJar") {
     dependencies {
         relocate("org.bstats", "com.plotsquared.holoplots.metrics")
         relocate("net.kyori.adventure", "com.plotsquared.core.configuration.adventure")
+        relocate("net.kyori.options", "com.plotsquared.core.configuration.options")
+        relocate("io.papermc.lib", "com.plotsquared.holoplots.paperlib")
```

Comment:
Do we need to shade paperlib into holoplots? PlotSquared shades paperlib and holoplots depends on PlotSquared, wouldn't `compileOnly` be enough in this case?

Reply:
I did have compileOnly at the beginning, which resulted in ClassNotFoundExceptions. Not sure if I can just define it again as compileOnly and relocate to the plot squared relocated package or something like that

Reply:
Oh, I'd assume that's down to the fact that P2 relocates paperlib. Given that's not API, it's not exposed and we're working with the default class path here, causing the ClassNotFoundExceptions, becuause at runtime there's no paperlib under that classpath.